smart-metastore/src/main/java/org/smartdata/metastore/dao/accesscount/AccessCountTableManager.java

```diff
@@ -151,44 +153,198 @@ private void recoverTables() {
     }
   }
 
-  private List<AccessCountTable> getTablesForLast(
-      long startTime, long endTime) throws MetaStoreException {
+  private List<AccessCountTable> getAccessCountTables(long startTime,
+                                                      long endTime) throws MetaStoreException {
+    return getAccessCountTables(startTime, endTime, null);
+  }
 
-    List<AccessCountTable> results = new ArrayList<>();
-    TimeGranularity searchIntervalGranularity = TimeGranularity.of(endTime - startTime);
+  private List<AccessCountTable> getAccessCountTables(long startTime,
```

Comment:
Please, add javadoc describing the overall algorithm used to find tables.

Reply:
done